.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.7.5
+3.1.2
```

Comment:
`rbenv` was complaining so I had to sync this up to the ruby version stated in the `Gemfile`